src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -72,15 +72,15 @@
             <ButtonWithIcon
                 :button-font-awesome-icon="['fas', 'arrow-up']"
                 :disabled="isFirstLayer"
-                :data-cy="`button-raise-order-layer-${id}`"
+                :data-cy="`button-lower-order-layer-${id}`"
```

Comment:
I would keep the data-cy naming here as it was and instead change the naming where it is used. Because even if technically you move the layer further back in an array, this will still result in raising the layer and not lowering it.